locales/pt_BR/pcsx2_Main.po

```diff
@@ -1282,7 +1282,7 @@ msgid ""
 "Note: Duplicate/Rename/Create/Delete will NOT be reverted with 'Cancel'."
 msgstr ""
 "\n"
-"Nota: Duplicar/Renomear/Criar/Excluir NÃO será revertido com \"Cancelar\"."
+"Nota: duplicar/renomear/criar/eliminar NÃO será revertido com "Cancelar"."
```

Comment:
Isn't "Duplicar/Renomear/Criar/Eliminar" better here, you think?

Reply:
I left in lowercase because the grammar says after a colon, the lowercase letter should be used. However, if you think it's best to leave as it is in English, that's okay, as this is not a big mistake.

Reply:
"Duplicar/Renomear/Criar/Eliminar" referes to the buttons, but are (here) also names of operations, and thus it's grammatically correct.

Reply:
Ok, let's use "Duplicar/Renomear/Criar/Eliminar then.